README.md

```diff
@@ -1,3 +1,36 @@
-Ironic Prometheus Exporter
-==========================
+# Ironic Prometheus Exporter #
 
+
+### Installation ###
+
+1 - Clone the repository in the machine where ironic is installed.
+```
+$ git clone https://github.com/metalkube/ironic-prometheus-exporter
+```
+2 - Install the driver (may require sudo permisions)
+```
+$ cd ironic-prometheus-exporter
+$ python setup.py install
+```
+3- Verify if the driver is installed
+```
+$ python verify_installation.py
+```
+Output in case of a successful instalation:
+`prometheus_exporter driver found.`
+Output in case of an unsuccessful instalation:
+`prometheus_exporter driver not found.`
+`Available drivers: ['log', 'messagingv2', 'noop', 'routing', 'test', 'messaging']`
+
+
+### Configuration ###
+
+After install the driver you will need to update the :ironic.conf: and add
+:file_path: option (the file extension should be .json)
+
+```
+[oslo_messaging_notifications]
+driver = prometheus_exporter
+transport_url = fake://
+file_path=/tmp/ironic_prometheus_exporter/metrics.json
```

Comment:
Ultimately a folder would be best because prometheus is only going to come along every five minutes or so.

Reply:
So the file name/format doesn't matter?

Reply:
nope, we should just be dumping content into a folder by some unique identifier in the message, per message and replacing any prior file.